eureka-client/src/main/java/com/netflix/discovery/CacheRefreshedEvent.java

```diff
@@ -6,6 +6,9 @@
  * 
  * @author brenuart
  */
-public class CacheRefreshedEvent extends DiscoveryEvent {
-
+public class CacheRefreshedEvent extends AbstractEurekaEvent {
```

Comment:
I updated the PR in a way that should be backwards compatible.  I'd rather not have two different event mechanisms unless absolutely necessary, especially since the older one can be built on top of the new one.  Let me know if the change addresses your backwards compatibility concerns.